fractal-puppy/Engine
# now the player can have a max navigation distance defined

To test:
- Select the Player game object and see that there is the "Max walking distance" field.
- Play the game and see that when you click far away from the player, it does not go too far.
